code/modules/clothing/suits/marine_armor/_marine_armor.dm

```diff
@@ -455,6 +477,17 @@
 	icon_state = "L6"
 	armor_variation = 0
 
+/obj/item/clothing/suit/storage/marine/light/Initialize(mapload)
+	. = ..()
+	if(SSticker.mode && MODE_HAS_FLAG(MODE_FACTION_CLASH))
+		armor_bullet = CLOTHING_ARMOR_MEDIUM
+		armor_bomb = CLOTHING_ARMOR_MEDIUMLOW
+		armor_internaldamage = CLOTHING_ARMOR_MEDIUMHIGH
+	else
+		armor_bullet = CLOTHING_ARMOR_MEDIUMLOW
+		armor_bomb = CLOTHING_ARMOR_MEDIUM
+		armor_internaldamage = CLOTHING_ARMOR_LOW
```

Comment:
```suggestion

```
Like that, the type is already defined with those stats, do that to every else you added